lib/Command/InstallDefaultFonts.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace OCA\Richdocuments\Command;
+
+use OCA\Richdocuments\Service\FontService;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class InstallDefaultFonts extends Command {
```

Comment:
Ah we still don't have a public API for commands 🙈 We can add the required classes as a stub to tests/stub.phpstub